output.json

```diff
@@ -0,0 +1,348 @@
+            {
+              "object_type": "Channel",
+              "channel_name": "Ex_639_Em_667",
+              "intended_measurement": null,
+              "detector": {
+                "object_type": "Detector config",
+                "device_name": "Camera",
+                "exposure_time": 2.0,
+                "exposure_time_unit": "millisecond",
+                "trigger_type": "Internal",
+                "compression": null
+              },
+              "additional_device_names": null,
+              "light_sources": [
+                {
+                  "object_type": "Laser config",
+                  "device_name": "Laser 639nm Left",
+                  "wavelength": 639,
+                  "wavelength_unit": "nanometer",
+                  "power": 90.0,
+                  "power_unit": "percent"
+                },
+                {
+                  "object_type": "Laser config",
+                  "device_name": "Laser 639nm Right",
+                  "wavelength": 639,
+                  "wavelength_unit": "nanometer",
+                  "power": 80.0,
+                  "power_unit": "percent"
+                }
+              ],
+              "variable_power": false,
+              "excitation_filters": null,
+              "emission_filters": [
+                {
+                  "object_type": "Device config",
+                  "device_name": "667/30"
+                }
+              ],
+              "emission_wavelength": 667,
+              "emission_wavelength_unit": "nanometer"
+            }
+          ],
+          "coordinate_system": {
+            "object_type": "Coordinate system",
+            "name": "SPIM_RPI",
+            "origin": "Origin",
+            "axes": [
+              {
+                "object_type": "Axis",
+                "name": "X",
+                "direction": "Left_to_right"
+              },
+              {
+                "object_type": "Axis",
+                "name": "Y",
+                "direction": "Anterior_to_posterior"
+              },
+              {
+                "object_type": "Axis",
+                "name": "Z",
+                "direction": "Superior_to_inferior"
+              }
+            ],
+            "axis_unit": "micrometer"
+          },
+          "images": [
+            {
+              "object_type": "Image spim",
+              "channel_name": "Ex_488_Em_525",
+              "dimensions_unit": "pixel",
+              "image_to_acquisition_transform": [
+                {
+                  "object_type": "Scale",
+                  "scale": [
+                    1.8,
+                    1.8,
+                    1.8
+                  ]
+                },
+                {
+                  "object_type": "Translation",
+                  "translation": [
+                    0.0,
+                    0.0,
+                    0.0
+                  ]
+                }
+              ],
+              "dimensions": null,
```

Comment:
isn't dimensions meant to be the scale? rather than having the scale in the transform?